message.go

```diff
@@ -33,18 +33,24 @@ type Message struct {
 
 // MessageSend stores all parameters you can send with ChannelMessageSendComplex.
 type MessageSend struct {
-	Content string        `json:"content"`
-	Tts     bool          `json:"tts"`
-	Embed   *MessageEmbed `json:"embed"`
-	Nonce   string        `json:"nonce"`
+	MessageEdit
+	Tts   bool          `json:"tts"`
+	Embed *MessageEmbed `json:"embed"`
 }
 
 // MessageEdit stores all parameters you can send with ChannelMessageSendComplex.
 type MessageEdit struct {
```

Comment:
Is there a better name here? Do we need the MessageEdit? What if we just document that Tts doesn't work when editing? In that way we could just have 'MessageParameters' as the only struct that both ComplexSend and ComplexEdit take.